src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -33,17 +37,7 @@ public class Conveyor extends SubsystemBase {
 
   /** False is blocked (i.e. there is a note) */
   public DigitalInput intakeBeamBreakSensor = new DigitalInput(RobotMap.INTAKE_BEAM_BREAK_DIO);
-
-  /**
-   * These aren't actually limit switches; we just use SparkLimitSwitch objects to access them
-   * easily. At the moment, only sensor one actually exists. Sensor one is the closest to the
-   * intake. True means blocked (there is a note). TODO: If we use these, add them to Shuffleboard.
-   */
-  SparkLimitSwitch
-      beamBreakSensorOne = frontMotor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen),
-      beamBreakSensorTwo = frontMotor.getReverseLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen),
-      beamBreakSensorThree = backMotor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen),
-      beamBreakSensorFour = backMotor.getReverseLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+  public DigitalInput frontConveyorBeamBreakSensor = new DigitalInput(RobotMap.FRONT_CONVEYOR_BEAM_BREAK_DIO);
```

Comment:
Add back. Do we know which is which?

Reply:
We've been using `beamBreakSensorOne`, which is the front-most on the conveyor so I think using `frontConveyorBeamBreakSensor` would be identical behavior. Three and four don't actually exist so I'm not gonna have them in the code for now, but I'll add back the back conveyor sensor.

Reply:
Done